symbology/custom-dictionary-style/README.md

```diff
@@ -10,27 +10,34 @@ When symbolizing geoelements in your map, you may need to convey several pieces
 
 ## How to use the sample
 
-Pan and zoom the map to see the symbology from the custom dictionary style.
+Use the radio buttons to toggle between the dictionary symbols from the web style and style file. Pan and zoom around the map to see the symbology from the chosen dictionary symbol style. The web style and style file are slightly different to each other to give a visual indication of the switch between the two.
 
 ## How it works
 
-1. Create a new `DictionarySymbolStyle` by passing in the path to the custom style file (.stylx).
-2. Create a new `DictionaryRenderer`, providing the dictionary symbol style.
-3. Apply the dictionary renderer to a feature layer using `layer.setRenderer(dictionaryRenderer)`.
+1. Create a `PortalItem`, referring to a `Portal` and the item ID of the web style.
+2. Based on the style selected:
+-  If the web style toggle has been selected, create a new `DictionarySymbolStyle` from the portal item using `new DictionarySymbolStyle(portalItem)`
+-  If the file style toggle has been selected, create a new dictionary symbol style using `DictionarySymbolStyle.createFromFile(stylxFile.getAbsolutePath())`
+3. Load the symbol dictionary asynchronously using `dictionarySymbolStyle.loadAsync()` and wait for it to completely load using `dictionarySymbolStyle.addDoneLoadingListener()`.
+4. Create a new `DictionaryRenderer`, providing the dictionary symbol style.
```

Comment:
The field overrides are used so that the symbols created from the web style are different from the symbols automatically rendered from the style file. 
```suggestion
4. To contrast the symbols rendered from the two `DictionarySymbolStyle`s, create a new `DictionaryRenderer` providing the dictionary symbol style, and manually map the attributes from the feature layer to those expected by the `DictionarySymbolStyle`.
```